CKAN-meta/build.sh

```diff
@@ -178,6 +115,137 @@ inject_metadata () {
     tar -czf master.tar.gz CKAN-meta-master
 }
 
+# ------------------------------------------------
+# Print the list of game versions we know about.
+# ------------------------------------------------
+get_versions() {
+    # Usage: VERSIONS=( $(get_versions) )
+
+    # Get our official list of releases
+    BUILDS_JSON=$(wget -q -O - https://raw.githubusercontent.com/KSP-CKAN/CKAN/master/Core/builds.json)
+
+    # Get just the MAJOR.MINOR.PATCH strings
+    echo $BUILDS_JSON | "$JQ_PATH" --raw-output '[.builds[] | sub("\\.[0-9]+$"; "")] | unique | reverse | .[]'
```

Comment:
Hmm, [`sub` was added to `jq` in 2014](https://github.com/stedolan/jq/blob/master/ChangeLog). Is the version used by these scripts really that old?

Reply:
I'm not sure, but the docker file that builds the container used by jenkins is in the repo.

Reply:
OK, I see "jessie" mentioned in the apt sources for mono, and [that version of Debian seems to have jq 1.4](https://packages.debian.org/search?keywords=jq). I downloaded that release, and passing it a command involving `sub` indeed gives me the same error, while jq 1.5 works fine.

Can we upgrade to a more recent OS?

Reply:
Well it's based off the jenkins slave image. So probably, but it would require ensuring all the jenkins bits work, though I'm pretty sure we're just using SSH + injecting the public key into the image as none of the variables etc actually worked.

Reply:
OK, in that case it's easy enough to eliminate the call to `sub`. See #39.